docs/docs/advanced/server-side-logic.md

```diff
@@ -1,25 +1,24 @@
 ---
-sidebar_position: 40
+sidebar_position: 100
 ---
 
-# Logic Restrictions
+# Server-Side Logic
 
-When writing multiplayer games in Rune, the game state is distributed across all players and a Rune server. To make sure all players see the same thing there are some limitations on what kind of code you can write in the `logic.js` file (`client.js` is not affected).
+Rune uses a server-authoritative approach to ensure games run smoothly and prevent cheating. To do this, your game's `logic.js` file will run on every client and on Rune's servers (see [Syncing Game State](../how-it-works/syncing-game-state) for more details). Rune ensures that all players see the same thing when playing your game by having some limitations on what kind of code you can write in your `logic.js` file.
 
 The primary aim is to ensure that the code is deterministic, meaning that if you run the code multiple times with the same input it will produce the same result. The main contributors to non-deterministic code in this context is use of other non-deterministic functions such as `Date.now()` and access of shared state such as counters and cache variables. **To avoid errors in production the Rune Multiplayer SDK will check your code** for the following unsafe patterns:
 
 - **Mutation and assignment of variables outside of current function scope**. Since mutation of function arguments are allowed for ergonomic reasons, it's also not allowed to read non-local variables, just invoking functions/methods.
-- **`async`/`await` syntax** as the SDK requires logic to be synchronous.
+- **`async`/`await` syntax** as logic must be synchronous.
 - **`try`/`catch` syntax** as this might interfere with Rune game control flow. Throwing is still allowed in case your program hits an irrecoverable error state.
 - **Use of non-deterministic runtime built-ins** such as `Date`, `fetch` etc.
 - **Modules** such as `import/export` and `require` as the SDK requires your logic to be self-contained in a single file. If you want to share logic or split the file into multiple you may use a bundler like rollup to produce a single file from many.
-- **Non-serializable data types** such as `Set`, `Map`, `Promise` etc because the game state needs to be able to be represented as `JSON`.
 - **Regular expressions** because they are stateful.
 - **Use of `eval`** because it's potentially harmful and can be used to bypass above rules.
 
 A notable exception to this list is `Math.random()` which is ensured to be deterministic, see [Randomness](randomness.md) to read more about how this works.
 
-The [Rune CLI](cli.md) will warn you if your game logic uses unsafe code so don't worry!
+The [Rune CLI](publishing/cli.md) will also warn you if your game logic uses unsafe code when uploading so don't worry.
```

Comment:
Not sure if it also makes sense to mention something along the lines of "not all flagged code is unsafe and not all unsafe code automatically flagged"

Reply:
Fixed in https://github.com/rune/rune-games-sdk/pull/175/commits/19263232dc0660e0d055db617ae9c01fac439646 👍